doc/ref_model/chapters/chapter03.md

```diff
@@ -98,17 +98,25 @@ _**Example**: a virtual compute descriptor as defined in TOSCA Simple Profile fo
 
 <a name="3.2.3"></a>
 ### 3.2.3 Storage
-A block device of a certain size for persisting information which can be created and dynamically attached to/detached from a virtual compute. A storage device resides in a tenant context and exists independently from any compute host.
 
-_**Example**: an OpenStack cinder volume._
+A workload can request different type of data availibility : persistent or ephemeral storage.
```

Comment:
availability is typically in terms of percentage

```suggestion
A workload can request different types of storage based on data longevity: persistent or ephemeral storage.
```

Reply:
@pgoyal01 Thanks, adopted!